src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -848,14 +848,15 @@ def cli(self, interface=None, output=None):
         interface_dict = {}
 
         #GigabitEthernet3/0/28: Ifindex = 36
-        p1 =  re.compile(r'(?P<interface_index>\S+) +Ifindex = +(?P<ifIndex>\d+)$')
+        #unrouted VLAN 1003: Ifindex = 7
+        p1 =  re.compile(r'^(unrouted\s+)?(?P<interface_index>(VLAN\s)?\S+) +Ifindex = +(?P<ifIndex>\d+)$')
 
         for line in out.splitlines():
             m = p1.match(line)
```

Comment:
We now put the example here too
```suggestion
            #GigabitEthernet3/0/28: Ifindex = 36
            #unrouted VLAN 1003: Ifindex = 7
            m = p1.match(line)
```